### PR TITLE
Streaming data fraud detection sample added

### PR DESCRIPTION
**Pull request by giedri**
*Issue #, if available:*

*Description of changes:*
Added one more sample - streaming data fraud detection.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: streaming-serverless-fraud-detection/README.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
+# Real-time transaction fraud detection/prevention using Serverless and Amazon Fraud Detector
+
+This example demonstrates a serverless approach to detect online transaction fraud in near real-time. It shows how detection can be plugged into various data streaming and event- driven architectures, depending on the outcome to be achieved and actions to be taken to prevent fraud - alert the customer/user about the fraud, flag the transaction for additional review, etc.
+
+This sample implements three architectures:
+ - [Streaming data inspection and fraud detection/prevention](#streaming-data-inspection-and-fraud-detectionprevention) - using Amazon Kinesis Data Stream, AWS Lambda, AWS Step Functions, and Amazon Fraud Detector
+ - [Streaming data enrichment for fraud detection/prevention](#streaming-data-enrichment) - using Amazon Kinesis Data Firehose, AWS Lambda, and Amazon Fraud Detector
+ - [Event data inspection and fraud detection/prevention](#event-data-inspection) - using Amazon EventBridge, AWS Step Functions, and Amazon Fraud Detector
+
+## Streaming data inspection and fraud detection/prevention
```
Comment (dancfox) [inline streaming-serverless-fraud-detection/README.md:10]: Is there a reason that you use the `detection/prevention` combination throughout? Perhaps this is industry specific terminology? If not can we just simplify to use `detection`?

Comment (dancfox) [reply streaming-serverless-fraud-detection/README.md:10]: So you can brush up on service name usage policy: 
https://w.amazon.com/bin/view/AWSDocs/editing/service-names

**Comment by dancfox** [comment]
Please add license terms to all `yaml` files.

**Comment by dancfox** [comment]
I notice the samconfig.toml is getting added to git. Can you fix the `.gitignore` file in the root directory and remove all the samconfig.toml files from your branch?

**Comment by dancfox** [comment]
I suggested some edits here: https://github.com/giedri/serverless-samples/pull/5.

File: streaming-serverless-fraud-detection/README.md (hunk 1 above)
Comment (giedri) [reply streaming-serverless-fraud-detection/README.md:10]: Service names updated

**Pull request merged**